new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/KubernetesExec.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.actions.impl.primitive;
+
+import java.io.IOException;
+
+import io.kubernetes.client.openapi.ApiException;
+
+/** A base class for an object which can execute a command in an Kubertenes containers. */
+public abstract class KubernetesExec {
+  private boolean stdin = true;
+  private boolean tty = true;
+
+  boolean isStdin() {
+    return stdin;
+  }
+
+  public void setStdin(boolean stdin) {
```

Comment:
fluent pattern does not use "set" prefix
also this field name is bad - i read "stdin" and i assume you mean the stdin - but you really mean something like "do you want to have a stdin" or something?? choose a better name